### DISCUSSION TITLE
2006/12/11/%E5%BE%AA%E7%8E%AF%E7%BB%93%E6%9E%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#136] [Announcements]
# 2006/12/11/%E5%BE%AA%E7%8E%AF%E7%BB%93%E6%9E%84

https://lv.qizhen.xyz/#docs/structure\for\loop

https://blog.qizhen.xyz/2006/12/11/%E5%BE%AA%E7%8E%AF%E7%BB%93%E6%9E%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：


帆
2007年05月8日 21:37 编辑
谢谢你的文章，希望能再多发些大家关心的问题，全力支持！！

回复
斌
2007年02月6日 08:41 编辑
收益非浅，非常感谢！ ：）

回复
清扬
2007年01月5日 20:31 编辑
很喜欢看你写的技术文章,对我很有启发.急切等待更新~
顺便问一句,你是NI的工程师吗?

回复
